src/Npgsql/MultiplexingDataSource.cs

```diff
@@ -380,19 +380,18 @@ protected override async ValueTask DisposeAsyncBase()
 
     struct MultiplexingStats
     {
-        internal Stopwatch Stopwatch;
+        internal long StartTimestamp;
         internal int NumCommands;
 
         internal void Reset()
         {
             NumCommands = 0;
-            Stopwatch.Reset();
```

Comment:
It seems like in #3918 I accidentally made it so we never start stopwatch at all...